server/index.js

```diff
@@ -72,6 +72,20 @@ app.use("/news/filter", (req, res, next) => {
     });
 });
 
+// Firebase config
```

Comment:
@victoriaj1003 Can you remove this file from your PR?

Reply:
Hi, I just tried running git rm -f server/index.js and then doing git push origin victoriaj1003\clearConsoleErrors but it looks like the file is still in my push.  Is there a better way to remove it?  Thanks.

Reply:
@victoriaj1003 you need to update you local branch with the file from master.
>>git checkout origin/master -- path/to/file

you can confirm the file was updated by checking the status of changes. it will show changed files
>>git status

complete the workflow with 
>>git add .
>>git commit -m 'undo changes to file xxx'
>>git push origin victoriaj1003clearConsoleErrors

Reply:
@reneefromhold I just pushed the server/index.js file again with the firebase api info deleted.  i'm not sure if that works for the pr.  Can you review it and let me know if it's right?  Thanks!!

Reply:
@victoriaj1003 It looks good now, thanks!  I still need to test locally, and will let you know once that is complete. Should be within the next couple of hours.